src/controllers/director.confirm.correspondence.address.controller.ts

```diff
@@ -28,11 +31,15 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
       returnPageUrl = urlUtils.getUrlToPath(DIRECTOR_CORRESPONDENCE_ADDRESS_MANUAL_PATH, req);
     } else {
       //edge case should not happen
-      returnPageUrl = req.headers.referer!
+      returnPageUrl = urlUtils.getUrlToPath(DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_CHOOSE_ADDRESS_PATH, req);
```

Comment:
should we redirect the user to the look-up page instead? The choose(array page) might end up erroring out of serviceAdress.postcode is null/undefined. 
I should have raised the same concern when reviewing the residential address confirm page.

Reply:
Yeah good catch! I didn't read it properly and just assumed that's where it was going